website/app/styles/pages/application/header.scss

```diff
@@ -150,15 +196,4 @@
   &:hover {
     color: var(--doc-color-white);
   }
-
```

Comment:
simplified the breakpoints logic, it was used only for the homepage/header, now everything has standard media queries.